internal/daemon/controller/handlers/targets/tcp/target_service_test.go

```diff
@@ -2697,6 +2700,8 @@ func TestAuthorizeSession(t *testing.T) {
 			gotCred.Secret = nil
 
 			got.AuthorizationToken, got.SessionId, got.CreatedTime = "", "", nil
+			// Expiration can't be determined ahead of time so just copy it
+			want.Expiration = got.Expiration
 			assert.Empty(t, cmp.Diff(got, want, protocmp.Transform()))
```

Comment:
Alternatively, just ignore this field
```suggestion
			assert.Empty(t, cmp.Diff(got, want, protocmp.Transform(), protocmp.IgnoreFields(&pb.SessionAuthorization{}, "expiration"))
```